packages/workflows/src/handlers/price-list/remove-prices.ts

```diff
@@ -0,0 +1,29 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IPricingModuleService } from "@medusajs/types"
+import { WorkflowArguments } from "../../helper"
+
+type Result = {
+  deleted: string[]
+}
+
+export async function removePrices({
+  container,
+  data,
+}: WorkflowArguments<{
+  moneyAmountIds: string[]
+}>): Promise<Result> {
+  const { moneyAmountIds } = data
+  const pricingService: IPricingModuleService = container.resolve(
+    ModuleRegistrationName.PRICING
+  )
+
+  await pricingService.deleteMoneyAmounts(moneyAmountIds)
+
+  return {
```

Comment:
todo later: return string[] as specified by the workflow return type. Check all workflows and end points

Reply:
already done